vega-settings

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env sh
+# Launch the Python settings app installed under share/vegastrike
```

Comment:
Q: how are we managing the Python dependencies for `vega-settings` such as `kivy`?